packages/components/src/systems/DoseCalculator/index.tsx

```diff
@@ -174,15 +174,15 @@ export default function DoseCalculator(props: DoseCalculatorProps) {
               type="number"
               inputmode="decimal"
               value={daysSupply()}
-              onInput={(e: Event) => setDaysSupply(e.currentTarget?.valueAsNumber)}
+              onInput={(e) => setDaysSupply(e.currentTarget?.valueAsNumber)}
             />
           </InputGroup>
           <InputGroup label="Doses per Day">
             <Input
               type="number"
               inputmode="decimal"
               value={dosesPerDay()}
-              onInput={(e: Event) => setDosesPerDay(e.currentTarget?.valueAsNumber)}
+              onInput={(e) => setDosesPerDay(e.currentTarget?.valueAsNumber)}
```

Comment:
ah, ya this works now because I fixed this upstream in `Input` https://github.com/Photon-Health/client/blob/0074160ec3ae562cd5e82fb6ddae290495e072b5/packages/components/src/particles/Input/index.tsx#L11